cogs/cmd_interface/leaderboard.py

```diff
@@ -25,12 +37,23 @@ async def _leaderboard_current(self, interaction: discord.Interaction):
         await interaction.followup.send(embed=embed, view=view)
         view.response = await interaction.original_response()
     
-    @rank_group.command(name = "previous", description = "Take a look at LLC's previous Hall of Fame")
-    async def _leaderboard_previous(self, interaction: discord.Interaction):
+    @rank_group.command(name="previous", description="Take a look at LLC's previous Hall of Fame")
+    @app_commands.describe(by_user="User sorted by role")
+    @app_commands.choices(by_user=[
+        app_commands.Choice(name="Leetcoder of the Month", value="Leetcoder of the Month"),
```

Comment:
Đoạn này nên viết gọn hơn dựa vào `role_emojies`, và sử dụng map để ánh xạ từng phần tử trong mảng này để ra 1 choice.

https://stackoverflow.com/questions/25082410/apply-function-to-each-element-of-a-list